indexing-hadoop/src/main/java/io/druid/indexer/IndexGeneratorJob.java

```diff
@@ -382,8 +383,18 @@ protected void reduce(
           context.progress();
           final InputRow inputRow = index.formatRow(parser.parse(value.toString()));
           allDimensionNames.addAll(inputRow.getDimensions());
-
-          int numRows = index.add(inputRow);
+          int numRows;
+          try {
+            numRows = index.add(inputRow);
+          } catch (ParseException e) {
+            if (config.isIgnoreInvalidRows()) {
+              log.info("Ignoring invalid row [%s] due to parsing error: %s", value.toString(), e.getMessage());
```

Comment:
Probably should make this a warn, also put the error as the first argument like this:
`log.info(e,"Ignoring invalid row [%s] due to parsing error", value.toString());`